docs/FAQ.md

```diff
@@ -0,0 +1,58 @@
+# Frequently asked questions
+
+### Why does my branch not move to the new commit after `jj new/commit`?
+
+Jujutsu's branches are akin to Git Tags, they always point to a single commit.
+To move them, use `jj branch set`. 
+
+### Where is my commit, why is it not visible in `jj log`? 
+
+If you don't find it with `jj log -r 'all()'`, it may be time to set the config
```

Comment:
I think you mean "If you do find...". I think this needs more details:

> Is your commit visible with `ll log -r 'all()'`?
>
> If yes, you should be aware that `jj log` only shows the revisions matching `revsets.log` by default. You can change it as described in [`config.md`] to show more revisions.
>
> If not, the revision may have been abandoned (usually with `jj abandon`). In that case, `jj log -r commit_id` should show the revision as "hidden". `jj new commit_id` or `jj edit commit_id` should make the revision visible again.

Reply:
This is text is great.

Reply:
done.